runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -3462,7 +3462,8 @@ remoteCompile(J9VMThread *vmThread, TR::Compilation *compiler, TR_ResolvedMethod
    // 1. Requests an AOT cache store or load from the server, or
    // 2. Accesses the JITServer AOT deserializer in any way.
    // That moment is currently right here, when we get the new known IDs that are cached in the deserializer.
-   ((TR::CompilationInfoPerThread *)compInfoPT)->clearDeserializerWasReset();
+   auto vm = TR_J9VMBase::get(compInfo->getJITConfig(), vmThread);
```

Comment:
Can't we get the frontend from the compInfoPT or something like that

Reply:
Had to add an API to `TR::CompilationInfoPerThreadBase` to return the front end, but I updated the code in the force push (had to rebase to get a successful local build).